### PR TITLE
On boarding flow

### PR DESCRIPTION
**Pull request by LinnierGames**
### What has Changed
- The user is logged in __and persisted__ **after** the user is taken from the login flow to the home tab bar

### What's New
- Logout from the SettingsVc
- OnBoardingDirector manages what view controller should be the next Vc depending on what the user has already accepted (location, verify phone number)

- [ ] Populate the images in the tutorial
- [ ] Shorten the descriptions of the tutorial

### CODE REVIEW

**Comment by LinnierGames** [comment]
[On-boarding Flow](https://trello.com/c/4yjGmfy1/31-on-boarding-flow)

File: KifuSF/KFCLocationServiceDisclaimer.swift [hunk 1]
```diff
@@ -62,13 +107,18 @@ extension KFCLocationServiceDisclaimer: UIConfigurable {
         view.backgroundColor = UIColor.Pallete.White
         
         title = "Location Privacy"
-        locationServiceDisclaimerLabel.text = "In order to use Kifu we will need to know you location only while using the app."
+        locationServiceDisclaimerLabel.text = """
```
Comment (pondorasti) [inline KifuSF/KFCLocationServiceDisclaimer.swift:104]: This could be handled in the configureData method of UIConfigurable.

**Comment by pondorasti** [comment]
I have noticed that there are some TODOs in KFCPhoneNumberValidation, what is the status of them? are they supposed to be finished in this PR?

**Review by pondorasti — APPROVED** [review]
Everything looks good! Maybe the TODOs could be fixed in a latter PR.

**Comment by LinnierGames** [comment]
@Pondorasti i've added the changes you've mentioned. If i could use your help with change the uilabel to a button for the noCodeLabel that would be great! I've added a `sendTextMessage()` method [here](https://github.com/Kifuapp/KifuSF/pull/35/files#diff-5473e2e0e26eea21c9ceef0e8d9d8913R91) to make it easier when setting up the button.

**Comment by LinnierGames** [comment]
@Pondorasti , @woodward4422 will be finishing the rest of the checkboxes in the RP description before I merge this in

**Pull request merged**